modules/nodes-descheduler-configuring-strategies.adoc

```diff
@@ -32,24 +32,31 @@ spec:
   strategies:
     - name: "LowNodeUtilization" <1>
       params:
-       - name: "cputhreshold"
+       - name: "CPUThreshold"
          value: "10"
-       - name: "memorythreshold"
+       - name: "MemoryThreshold"
          value: "20"
-       - name: "podsthreshold"
+       - name: "PodsThreshold"
          value: "30"
-       - name: "memorytargetthreshold"
+       - name: "MemoryTargetThreshold"
          value: "40"
-       - name: "cputargetthreshold"
+       - name: "CPUTargetThreshold"
          value: "50"
-       - name: "podstargetthreshold"
+       - name: "PodsTargetThreshold"
          value: "60"
-       - name: "nodes"
+       - name: "NumberOfNodes"
          value: "3"
     - name: "RemoveDuplicates" <2>
+    - name: "RemovePodsHavingTooManyRestarts" <3>
+      params:
+       - name: "PodRestartThreshold"
+         value: "10"
+       - name: "IncludeInitContainers"
+         value: "false"
 ----
-<1> The `LowNodeUtilization` strategy provides additional parameters, such as `cputhreshold` and `memorythreshold`, that you can optionally configure.
+<1> The `LowNodeUtilization` strategy provides additional parameters, such as `CPUThreshold` and `MemoryThreshold`, that you can optionally configure.
 <2> The `RemoveDuplicates`, `RemovePodsViolatingInterPodAntiAffinity`, `RemovePodsViolatingNodeAffinity`, and `RemovePodsViolatingNodeTaints` strategies do not have any additional parameters to configure.
+<3> The `RemovePodsHavingTooManyRestarts` strategy has two parameters, `PodRestartThreshold` and `IncludeInitContainers`, that you can optionally configure.
```

Comment:
The `PodRestartThreshold` param is actually required to use `RemovePodsHavingTooManyRestarts`